IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricModelResource/Entities/IfcSectionedSurface/Documentation.md

```diff
@@ -1,15 +1,16 @@
-A kind of surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the CrossSections between CrossSectionPositions; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
-The CrossSections are oriented with the Y axis of each profile facing upwards in +Z direction or vertically perpendicular to the Directrix, depending on the Attribute FixedAxisVertical value.
+A surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the _CrossSections_ between _CrossSectionPositions_; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
+
+The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile X axis is oriented perpendicularly to the left of the _Directrix_ (same direction as _RefDirection_ derived in _IfcAxis2PlacementLinear_) as facing forward along the directrix, and the profile Y axis is oriented upwards according to _Axis_ attribute of _IfcAxis2PlacementLinear_.
```

Comment:
This up, left etc. is problematic. I would propose to avoid it entirely.
```suggestion
The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile Y axis is oriented according to _Axis_ attribute of _IfcAxis2PlacementLinear_ and the X is derived from the cross product of _Directrix_ and the _Axis_ attribute of _IfcAxis2PlacementLinear_.
```

Reply:
I agree with removal of up left etc.

However, I wonder if it's not better to use Axis1 Axis2 and Axis3 as defined by the IfcAxis2Placements?